libs/common/build.gradle

```diff
@@ -21,6 +21,10 @@ dependencies {
   // This dependency is used only by :libs:core for null-checking interop with other tools
   compileOnly "com.google.code.findbugs:jsr305:3.0.2"
 
+  compileOnly "org.bouncycastle:bc-fips:${versions.bouncycastle_jce}"
+  compileOnly "org.bouncycastle:bcutil-fips:${versions.bouncycastle_util}"
+  api "org.bouncycastle:bcpkix-fips:${versions.bouncycastle_pkix}"
```

Comment:
I think this is likely to be controversial because it violates the comment yelling at us three lines below. We'll either have to revisit the "!!!! no third party dependencies !!!!" policy and relax it to allow this bouncy castle dependency, or else find a way to architect this where BC can be supplied at runtime (via an SPI or something like that).

@reta what do you think?

Reply:
> @reta what do you think?

💯 , this dependency should not be in `common` which is shared between mostly any other OpenSearch component